caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -117,6 +117,7 @@ class SqlEditorTopToolbar extends React.Component {
   }
   render() {
     const tables = this.props.tables.filter((t) => (t.queryEditorId === this.props.queryEditor.id));
+    const shouldShowReset = window.location.search === '?reset=1';
```

Comment:
sql-lab isn't using url params for anything else currently. i think this will probably be a short term thing, so it can be toggled for a demo tomorrow. we should remove it once most of the dev work is done.